### PR TITLE
Update Gitter bridge section to point to the new native bridge

### PR DESCRIPTION
**Pull request by MadLittleMods**
Update [Gitter bridge section](https://matrix.org/bridges/) to point to the new native bridge

Fix https://github.com/matrix-org/matrix.org/issues/903

### CODE REVIEW

File: gatsby/content/projects/2016-02-10-matrix-appservice-gitter.mdx [hunk 1]
```diff
@@ -14,6 +14,8 @@ bridges: Gitter
 featured: true
 ---
 
+**Note:** This project is now deprecated in favor of the native Gitter bridge
+
 This project bridges [Gitter](https://gitter.im) to Matrix, via the AS API on the Matrix side, and a Gitter user on the Gitter side.
 
 Get it from [github](https://github.com/matrix-org/matrix-appservice-gitter).
```
Comment (MadLittleMods) [inline gatsby/content/projects/2016-02-10-matrix-appservice-gitter.mdx:21]: What is the difference between the gatsby and jekyll stuff?

I see this information is also at [`jekyll/_posts/projects/2018-11-20-bridges.md#gitter`](https://github.com/matrix-org/matrix.org/blob/a05cdebec41c24214cdacfdce0b171abaf451195/jekyll/_posts/projects/2018-11-20-bridges.md#gitter) but I am not sure what it corresponds to.

Comment (benparsons) [reply gatsby/content/projects/2016-02-10-matrix-appservice-gitter.mdx:21]: The jekyll content is not used in the build, we should remove it but there might be some legacy value...

**Review by benparsons — APPROVED** [review]
lgtm

File: gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+This project bridges [Gitter](https://gitter.im) to Matrix and is already hosted for you. It creates portal rooms for every public room on Gitter which you can join via `#org_repo:gitter.im` (just replace the `/` in the Gitter URI with an underscore `_`). For example https://gitter.im/nodejs/node corresponds to [#nodejs_node:gitter.im]](https://matrix.to/#/#nodejs_node:gitter.im).
+
+If you're already using [Element](https://element.io), you can browse all of the Gitter rooms in the public room directory (switch to the `gitter.im` server).
+
+✓ Public Chats  
+✗ Private Chats  
+✗ 1:1 Chats  
+✓ Native feeling usernames/avatar on both sides (virtual users)
+✓ Replies / Threaded conversations
```
Comment (MadLittleMods) [inline gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:25]: This slash is causing formatting issues. Is there a way to workaround this with the slash?

Comment (MadLittleMods) [reply gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:25]: Maybe just get rid of the space between like the `usernames/avatars above

File: gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+description: Bridges Gitter to the Matrix network
+author: Gitter
+maturity: Released
+language: JavaScript
+license: Apache-2.0
+repo: https://gitlab.com/gitterHQ/webapp
+thumbnail: /docs/projects/images/gitter-logo.svg
+bridges: Gitter
+featured: true
+---
```
Comment (MadLittleMods) [inline gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:15]: Is there a way to put this project above `matrix-appservice-gitter` on https://matrix.org/bridges/#gitter?

Do we need to make the date newer for `matrix-appservice-gitter` to sort it last?

Comment (MadLittleMods) [reply gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:15]: Maybe mark it as `featured: false` so it doesn't appear anymore?

File: gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx (hunk 2 above)
Comment (MadLittleMods) [reply gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:25]: Fixed in https://github.com/matrix-org/matrix.org/pull/922

File: gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx (hunk 3 above)
Comment (MadLittleMods) [reply gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:15]: Addressing in https://github.com/matrix-org/matrix.org/pull/922

**Comment by MadLittleMods** [comment]
Thanks for the review @benparsons 🐔

I've followed up in https://github.com/matrix-org/matrix.org/pull/922 to fix some formatting issues.

**Pull request merged**